src/utils/theme.ts

```diff
@@ -93,4 +93,10 @@ export const themeColorCss = css`
     --rgb-state-lock-locked: var(--mush-rgb-state-lock-locked, var(--rgb-green));
     --rgb-state-lock-unlocked: var(--mush-rgb-state-lock-unlocked, var(--rgb-red));
     --rgb-state-lock-pending: var(--mush-rgb-state-lock-pending, var(--rgb-orange));
+
+    /* State cover colors */
+    --rgb-state-cover-open: var(--mush-rgb-state-cover-open, var(--rgb-state-cover));
+    --rgb-state-cover-opening: var(--mush-rgb-state-cover-opening, var(--rgb-state-cover));
+    --rgb-state-cover-closed: var(--mush-rgb-state-cover-closed, var(--rgb-disabled));
+    --rgb-state-cover-closing: var(--mush-rgb-state-cover-closing, var(--rgb-disabled));
```

Comment:
```suggestion
    --rgb-state-cover-open: var(--mush-rgb-state-cover-open, var(--rgb-state-cover));
    --rgb-state-cover-closed: var(--mush-rgb-state-cover-closed, var(--rgb-disabled));
```
No need to have opening and closing color.

Reply:
Correct, forgot to remove these.